flytestdlib/storage/storage.go

```diff
@@ -78,6 +78,9 @@ type RawStore interface {
 	// Head gets metadata about the reference. This should generally be a light weight operation.
 	Head(ctx context.Context, reference DataReference) (Metadata, error)
 
+	// GetItems retrieves the paths of all items from the Blob store or an error
+	GetItems(ctx context.Context, reference DataReference) ([]string, error)
```

Comment:
would this be more accurately named ListItems?  Also what is retrieved?  The relative path to the reference input?  can we add comment?